srcpkgs/cni-plugins/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'cni-plugins'
+pkgname=cni-plugins
+version=0.7.4
+revision=1
+wrksrc="plugins-${version}"
+build_style=go
+go_import_path="github.com/containernetworking/plugins"
+short_desc="Container Network Interface (plugins)"
+maintainer="Cameron Nemo <camerontnorman@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/containernetworking/plugins"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=d47f2bda2bfd6699843a98d91a2e5c40aa58eacd3f2393c7696c158c410aad3c
+
+do_build() {
+	./build.sh
```

Comment:
Is there a reason to favor this build over an xbps-src mechanism?

Reply:
Does the list of plugins change regularly?

Reply:
There are 14 of them, so it is certainly possible that new ones are added (or the existing ones re-ordered). I would say it is more likely than the configuration version changing.